### PR TITLE
docs: add v0.9 docs

### PR DESCRIPTION
**Pull request by andrewrynhard**
Adds documentation for v0.9, copied from v0.8.

Signed-off-by: Andrew Rynhard <andrew@rynhard.io>

### CODE REVIEW

File: website/content/docs/v0.9/Guides/configuring-network-connectivity.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    <th class="px-4 py-2">Purpose</th>
+    <th class="px-4 py-2">Used By</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td class="border px-4 py-2">TCP</td>
+      <td class="border px-4 py-2">Inbound</td>
+      <td class="border px-4 py-2">50001*</td>
+    <td class="border px-4 py-2"><a href="https://talos.dev/docs/v0.8/learn-more/components/#trustd">trustd</a></td>
```
Comment (smira) [inline website/content/docs/v0.9/Guides/configuring-network-connectivity.md:64]: should we update this and many similar links to 0.9? and same for installer version tags, etc.

**Review by smira — COMMENTED** [review]
should we update Dockerfile to output to 0.9?

**Review by Unix4ever — APPROVED** [review]

**Comment by smira** [comment]
update gridsome config for the new sections

**Pull request merged**